CHANGELOG.md

```diff
@@ -6,6 +6,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
 
 ## [[Unreleased]]
+- When connected to nft devnet, hooks v2 testnet, or amm devnet, generate_faucet_wallet now defaults to using the faucet instead of requiring specification
```

Comment:
```suggestion
## [[Unreleased]]

### Fixed
- When connected to nft devnet, hooks v2 testnet, or amm devnet, generate_faucet_wallet now defaults to using the faucet instead of requiring specification
```